### PR TITLE
wip(openrtb): update IAB categories

### PR DESCRIPTION
**Pull request by darynl**
Add all IAB categories.

### CODE REVIEW

File: openrtb/category.go [hunk 1]
```diff
@@ -5,24 +5,396 @@ type Category string
...
-	CategoryIllegalContent      Category = "IAB26"
+	CategoryArtsAndEntertainment           Category = "IAB1"
+	CategoryBooksAndLiterature             Category = "IAB1-1"
+	CategoryCelebrityFanGossip             Category = "IAB1-2"
+	CategoryFineArt                        Category = "IAB1-3"
+	CategoryHumor                          Category = "IAB1-4"
+	CategoryMovies                         Category = "IAB1-5"
+	CategoryMusic                          Category = "IAB1-6"
+	CategoryTelevision                     Category = "IAB1-7"
+	CategoryAutomotive                     Category = "IAB2"
```
Comment (garukun) [inline openrtb/category.go]: Nitpick: Maybe we can divide the sections based on the IAB value? E.g. all `IAB1*` in one section, and then `IAB2*`, etc.

**Comment by garukun** [comment]
Commit title has wrong, category, should not be `wip(...)`. One small nitpick about organizing the code. Otherwise, LGTM

File: openrtb/category.go [hunk 2]
```diff
@@ -5,24 +5,396 @@ type Category string
...
+	CategoryCertifiedPreOwned              Category = "IAB2-5"
+	CategoryConvertible                    Category = "IAB2-6"
+	CategoryCoupe                          Category = "IAB2-7"
+	CategoryCrossover                      Category = "IAB2-8"
+	CategoryDiesel                         Category = "IAB2-9"
+	CategoryElectricVehicle                Category = "IAB2-10"
+	CategoryHatchback                      Category = "IAB2-11"
+	CategoryHybrid                         Category = "IAB2-12"
+	CategoryLuxury                         Category = "IAB2-13"
+	CategoryMiniVan                        Category = "IAB2-14"
```
Comment (hwong) [inline openrtb/category.go]: nitpick: lowercase v

File: openrtb/category.go [hunk 3]
```diff
@@ -5,24 +5,396 @@ type Category string
...
+	CategoryUSMilitary                     Category = "IAB4-10"
+	CategoryCareerAdvice                   Category = "IAB4-11"
+	CategoryEducation                      Category = "IAB5"
+	Category712Education                   Category = "IAB5-1"
+	CategoryAdultEducation                 Category = "IAB5-2"
+	CategoryArtHistory                     Category = "IAB5-3"
+	CategoryColledgeAdministration         Category = "IAB5-4"
+	CategoryCollegeLife                    Category = "IAB5-5"
+	CategoryDistanceLearning               Category = "IAB5-6"
+	CategoryEnglishasa2ndLanguage          Category = "IAB5-7"
```
Comment (hwong) [inline openrtb/category.go]: nitpick: CategoryEnglishAsA2ndLanguage

File: openrtb/category.go [hunk 4]
```diff
@@ -5,24 +5,396 @@ type Category string
...
+	CategoryPrivateSchool                  Category = "IAB5-13"
+	CategorySpecialEducation               Category = "IAB5-14"
+	CategoryStudyingBusiness               Category = "IAB5-15"
+	CategoryFamilyAndParenting             Category = "IAB6"
+	CategoryAdoption                       Category = "IAB6-1"
+	CategoryBabiesAndToddlers              Category = "IAB6-2"
+	CategoryDaycarePreSchool               Category = "IAB6-3"
+	CategoryFamilyInternet                 Category = "IAB6-4"
+	CategoryParentingK6Kids                Category = "IAB6-5"
+	CategoryParentingteens                 Category = "IAB6-6"
```
Comment (hwong) [inline openrtb/category.go]: nitpick: Teens

File: openrtb/category.go [hunk 5]
```diff
@@ -5,24 +5,396 @@ type Category string
...
+	CategoryIncontinence                   Category = "IAB7-29"
+	CategoryInfertility                    Category = "IAB7-30"
+	CategoryMensHealth                     Category = "IAB7-31"
+	CategoryNutrition                      Category = "IAB7-32"
+	CategoryOrthopedics                    Category = "IAB7-33"
+	CategoryPanicAnxietyDisorders          Category = "IAB7-34"
+	CategoryPediatrics                     Category = "IAB7-35"
+	CategoryPhysicalTherapy                Category = "IAB7-36"
+	CategoryPsychologyPsychiatry           Category = "IAB7-37"
+	CategorySenorHealth                    Category = "IAB7-38"
```
Comment (hwong) [inline openrtb/category.go]: nitpick: Senior

File: openrtb/category.go [hunk 6]
```diff
@@ -5,24 +5,396 @@ type Category string
...
+	CategoryCajunCreole                    Category = "IAB8-3"
+	CategoryChineseCuisine                 Category = "IAB8-4"
+	CategoryCocktailsBeer                  Category = "IAB8-5"
+	CategoryCoffeeTea                      Category = "IAB8-6"
+	CategoryCuisineSpecific                Category = "IAB8-7"
+	CategoryDessertsAndBaking              Category = "IAB8-8"
+	CategoryDiningOut                      Category = "IAB8-9"
+	CategoryFoodAllergies                  Category = "IAB8-10"
+	CategoryFrenchCuisine                  Category = "IAB8-11"
+	CategoryHealthLowfatCooking            Category = "IAB8-12"
```
Comment (hwong) [inline openrtb/category.go]: nitpick: Fat

File: openrtb/category.go [hunk 7]
```diff
@@ -5,24 +5,396 @@ type Category string
...
+	CategoryAppliances                     Category = "IAB10-1"
+	CategoryEntertaining                   Category = "IAB10-2"
+	CategoryEnvironmentalSafety            Category = "IAB10-3"
+	CategoryGardening                      Category = "IAB10-4"
+	CategoryHomeRepair                     Category = "IAB10-5"
+	CategoryHomeTheater                    Category = "IAB10-6"
+	CategoryInteriorDecorating             Category = "IAB10-7"
+	CategoryLandscaping                    Category = "IAB10-8"
+	CategoryRemodelingAndConstruction      Category = "IAB10-9"
+	CategoryLawGovtAndPolitics             Category = "IAB11"
```
Comment (hwong) [inline openrtb/category.go]: nitpick: Government

File: openrtb/category.go [hunk 8]
```diff
@@ -5,24 +5,396 @@ type Category string
...
+	CategoryBodyArt                        Category = "IAB18-2"
+	CategoryFashion                        Category = "IAB18-3"
+	CategoryJewelry                        Category = "IAB18-4"
+	CategoryClothing                       Category = "IAB18-5"
+	CategoryAccessories                    Category = "IAB18-6"
+	CategoryTechnologyAndComputing         Category = "IAB19"
+	Category3DGraphics                     Category = "IAB19-1"
+	CategoryAnimation                      Category = "IAB19-2"
+	CategoryAntivirusSoftware              Category = "IAB19-3"
+	CategoryCC                             Category = "IAB19-4"
```
Comment (hwong) [inline openrtb/category.go]: CategoryCCPlusPlus

File: openrtb/category.go [hunk 9]
```diff
@@ -5,24 +5,396 @@ type Category string
...
+	CategoryPornography                    Category = "IAB25-3"
+	CategoryProfaneContent                 Category = "IAB25-4"
+	CategoryHateContent                    Category = "IAB25-5"
+	CategoryUnderConstruction              Category = "IAB25-6"
+	CategoryIncentivized                   Category = "IAB25-7"
+	CategoryIllegalContent                 Category = "IAB26"
+	CategoryIllegalContent2                Category = "IAB26-1"
+	CategoryWarez                          Category = "IAB26-2"
+	CategorySpywareMalware                 Category = "IAB26-3"
+	CategoryInfringement                   Category = "IAB26-4"
```
Comment (hwong) [inline openrtb/category.go]: CategoryCopyrightInfringement

File: openrtb/category.go (hunk 1 above)
Comment (hwong) [reply openrtb/category.go]: 👍

**Comment by hwong** [comment]
minor nitpicks otherwise LGTM

**Comment by garukun** [comment]
LGTM

**Pull request merged**